Madam Q

JamesVanBoxtel opened the pull request:
Added Madam Q with all assets and metadata

JamesVanBoxtel committed:
Madam Q

Added Madam Q with all assets and metadata

Endaris commented on client/assets/default_data/characters/Madam Q/combo7_3.ogg:
Should be renamed to combo7_2

Endaris commented on client/assets/default_data/characters/Madam Q/chain2_3.ogg:
Not sure about the attack on this one, it sounds a little like there could be a sliver of the start missing, like someone forgot to breathe.

Endaris commented on client/assets/default_data/characters/Madam Q/chain11.ogg:
This also sounds like the start was cut a bit short.

Endaris commented on client/assets/default_data/characters/Madam Q/chain14_2.ogg:
This also sounds like the start was cut a bit short.

Endaris commented on client/assets/default_data/characters/Madam Q/garbage_land6.ogg:
This also sounds like the start was cut a bit short.

Endaris commented on client/assets/default_data/characters/Madam Q/garbage_land10.ogg:
This also sounds like the start was cut a bit short.

Endaris commented on client/assets/default_data/characters/Madam Q/chain5_3.ogg:
Not sure how much you can still do but this sound along with chain3, chain3_2 and garbage_land10 has the t breathout being relatively overwhelming to the rest of the SFX.
The other chain5 also suffer of this to some degree but on this one it stands out by far the most.

Endaris commented on client/assets/default_data/characters/Madam Q/taunt_down8.ogg:
It sounds a bit weird how she's getting stuck on the initial s for so long so I'd maybe take this one out after all.

JamesVanBoxtel replied:
Faded in the "so" that way its more fluid

Endaris commented on client/assets/default_data/characters/Madam Q/taunt_down11.ogg:
This sounds like it may cut off early.

Endaris commented on client/assets/default_data/characters/Madam Q/win10.ogg:
Not sure but on repeated listens I feel like the delivery on this is quite a bit weaker than win8 and win9 so I'd consider taking it out.

JamesVanBoxtel replied:
Okay

Endaris commented on client/assets/default_data/characters/Madam Q/telegraph/attack@2x.png_original:
This looks like an accidental addition.

Endaris reviewed (commented):
Did a relisten on the SFX with some more comments.
We didn't really do this for our previous characters but given that our Madam Q voice actor did try a certain diversity in delivery we may want to go ahead pick only one if the same line was delivered in drastically different ways.
This wasn't as necessary for our previous characters but I think in this case the material warrants it.

The telegraph should be color adjusted to match her garbage color, currently it's the Rattles color.

I'm still not entirely sold on the face/doubleface, I'll try to deliver on my promise that I'd try to apply Donavin's suggestions to the mask + fan combination.

JamesVanBoxtel commented:
I think we should just remove specific sounds that we think don't work like we have been doing, not remove every take but one for each line.

updated the telegraph

I'm okay doing double face stuff but don't know if we need to hold up this PR for it. Once we are happy with what we have decided I can redo the vector and land it as a second PR

JamesVanBoxtel committed:
Madam Q Updates

Adjusted chain 2, chain 2_2, and chain2_3 fades

Lowered T end of chain 5_3 chain3, chain3_2, garbage_land10, chain 5, chain5_2

Deleted win10, moved win19 -> 10

Moved to red telegraph art

reapplied metadata

Endaris commented on client/assets/default_data/characters/Madam Q/pop@2x.png:
Sorry for commenting on this so late after you reverted the color shift on this based on my request.
To me it seems like the pop frame goes a bit under when clearing chain garbage because of how closely the overall coloration and texture of the pop frame matches the garbage itself. Both Esme and Rattles solve that by the color of their pop symbol being clearly different from their filler. For Magellan the background texture is removed. I think that might work here as well.

Can wait until also reassessing face / doubleface however.

JamesVanBoxtel replied:
Sure I can do that when we reassess the face, but probably doesn't need to hold up landing?

Endaris committed:
shift combo 7_3 to combo 7_2
remove non-metadata version of attack sprite

Endaris commented on client/assets/default_data/characters/Madam Q/burst@2x.png:
This is missing its metadata.
I'd add it myself but I haven't quite been able to gather whether you or Donavin made the fan.
COPYING-ASSETS suggests it was you but the chat history in the thread suggests Donavin made it so...

JamesVanBoxtel replied:
I made the original, donavin made this version, I cleaned it up into this final version. I guess he would have the main copyright and I would be a contributor?

Endaris committed:
remove history metadata from telegraph and add entry for xmpMM metadata removal to CONTRIBUTING.md

Endaris commented on client/assets/default_data/characters/Madam Q/telegraph/attack@2x.png:
Copyright year in this should probably be 2025?

JamesVanBoxtel replied:
Yes

Endaris reviewed (commented):
Sounds are in the clear.
I added a new `-XMP-xmpMM:all=  ` to the contribution doc as some of the telegraph files had history metadata of file saves that made up more than half the file size and cleared them out for Madam Q. Did not recheck for the other characters.

We're almost there!

Endaris committed:
update metadata and COPYING-ASSETS with fixed information
